app/services/decode_service.rb

```diff
@@ -0,0 +1,25 @@
+module DecodeService
+	def self.attach_image(file, target)
+		image = Rails.env.test? ? file : split_base64(file)
+
```

Comment:
indentation

Reply:
yet again, github has changed the indentation to 8, indenting is normal and set to 2 on my vscode...

Reply:
not sure why this keeps happening